src/test/java/seedu/storemando/logic/parser/ListCommandParserTest.java

```diff
@@ -18,29 +15,4 @@ public void parse_emptyArg_throwsParseException() {
         ListCommand expectedListCommand = new ListCommand();
         assertParseSuccess(parser, "     ", expectedListCommand);
     }
-
-    @Test
-    public void parse_validArgs_nonGenericListReturnsListCommand() {
```

Comment:
Could add some tests since you removed these :)

Reply:
Agree with kums!